src/kernel/imfilter_cpu.jl

```diff
@@ -0,0 +1,11 @@
+# ------------------------------------------------------------------
+# Licensed under the MIT License. See LICENCE in the project root.
+# ------------------------------------------------------------------
+
+const array_kernel(array, ::CPUMethod) = array
+
+const view_kernel(array, I, ::CPUMethod) = view(array, I)
+
+function imfilter_kernel(img, krn, ::CPUMethod)
+  imfilter(img, centered(krn), Inner(), Algorithm.FFT())
+end
```

Comment:
I think we need to pass the KernelMethod as the first argument in all kernels. For example, CPUMethod should be the first argument by convention.